Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs

```diff
@@ -836,13 +836,7 @@ private void DetermineTargetBounds()
             }
             else
             {
-                // first remove old collider if there is any so we don't accumulate any 
```

Comment:
Will we "accumulate any box padding on consecutive calls of this method" under the new approach?

Reply:
Nope, no other colliders are added to the gameobject now beyond an initial one, so there is no need to continuously add/remove them